provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/OrganizationApi.java

```diff
@@ -1,36 +1,98 @@
 package gov.va.api.health.providerdirectory.api;
 
-import gov.va.api.health.providerdirectory.api.resources.Location;
+import gov.va.api.health.providerdirectory.api.datatypes.Identifier;
+import gov.va.api.health.providerdirectory.api.resources.OperationOutcome;
+import gov.va.api.health.providerdirectory.api.resources.Organization;
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.Parameter;
 import io.swagger.v3.oas.annotations.enums.ParameterIn;
 import io.swagger.v3.oas.annotations.media.Content;
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 
+import javax.ws.rs.DefaultValue;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 
 public interface OrganizationApi {
     @Operation(
-            summary = "Location Read",
+            summary = "Organization Read",
             description =
-                    "location link",
-            tags = {"Location"}
+                    "http://www.fhir.org/guides/argonaut/pd/StructureDefinition-argo-organization.html",
+            tags = {"Organization"}
     )
     @GET
-    @Path("Location/{id}")
+    @Path("Organization/{id}")
     @ApiResponse(
             responseCode = "200",
             description = "Record found",
             content =
             @Content(
                     mediaType = "application/json+fhir",
-                    schema = @Schema(implementation = Location.class)
+                    schema = @Schema(implementation = Organization.class)
             )
     )
-    Location LocationRead(
+    @ApiResponse(
+            responseCode = "400",
+            description = "Not found",

```

Comment:
400 is bad request.

404 is not found.

Reply:
I agree